File: src_python/4_nip_asn.py
nip = input("Masukkan NIP ASN: ")

tahun = nip[0:4]
bulan = int(nip[4:6])
tanggal = nip[6:8]

if bulan == 1:
    nama_bulan = "Januari"
elif bulan == 2:
    nama_bulan = "Februari"
elif bulan == 3:
    nama_bulan = "Maret"
elif bulan == 4:
    nama_bulan = "April"
elif bulan == 5:
    nama_bulan = "Mei"
elif bulan == 6:
    nama_bulan = "Juni"
elif bulan == 7:
    nama_bulan = "Juli"
elif bulan == 8:
    nama_bulan = "Agustus"
elif bulan == 9:
    nama_bulan = "September"
elif bulan == 10:
    nama_bulan = "Oktober"
elif bulan == 11:
    nama_bulan = "November"
elif bulan == 12:
    nama_bulan = "Desember"
else:
    print("Bulan tidak valid")
    exit()

print("Tanggal lahir ASN adalah",
      tanggal, nama_bulan, tahun)

File: src_R/4_nip_asn.R
nip <- readline("Masukkan NIP ASN: ")

tahun <- substr(nip, 1, 4)
bulan <- as.numeric(substr(nip, 5, 6))
tanggal <- substr(nip, 7, 8)

if (bulan == 1) {
  nama_bulan <- "Januari"
} else if (bulan == 2) {
  nama_bulan <- "Februari"
} else if (bulan == 3) {
  nama_bulan <- "Maret"
} else if (bulan == 4) {
  nama_bulan <- "April"
} else if (bulan == 5) {
  nama_bulan <- "Mei"
} else if (bulan == 6) {
  nama_bulan <- "Juni"
} else if (bulan == 7) {
  nama_bulan <- "Juli"
} else if (bulan == 8) {
  nama_bulan <- "Agustus"
} else if (bulan == 9) {
  nama_bulan <- "September"
} else if (bulan == 10) {
  nama_bulan <- "Oktober"
} else if (bulan == 11) {
  nama_bulan <- "November"
} else if (bulan == 12) {
  nama_bulan <- "Desember"
} else {
  cat("Bulan tidak valid\n")
  quit()
}

cat("Tanggal lahir ASN adalah",
    tanggal, nama_bulan, tahun, "\n")
